src/CMakeLists.txt

```diff
@@ -244,6 +244,18 @@ IF(LMMS_BUILD_WIN32)
 				DESTINATION .)
 	ENDIF()
 
+
+	SET(JACK_INSTALLER_VERSION "1.9.11")
+	# Bundle jack
+	IF (WANT_JACK)
+		ADD_CUSTOM_TARGET(jack_binary_installer ALL
+			COMMAND "curl" -OL "https://github.com/jackaudio/jackaudio.github.com/releases/download/${JACK_INSTALLER_VERSION}/Jack_v${JACK_INSTALLER_VERSION}_32_setup.exe"
```

Comment:
Instead of using `curl` command, you can use `file(DOWNLOAD ...)` command provided by CMake.